python/paddle/fluid/contrib/slim/tests/slim_int8_mkldnn_post_training_quantization.md

```diff
@@ -0,0 +1,82 @@
+# SLIM Post-training quantization (INT8 MKL-DNN)
+
+This document describes how to use [Paddle Slim](https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/fluid/advanced_usage/paddle_slim/paddle_slim.md) to convert a FP32 ProgramDesc with FP32 weights to an INT8 ProgramDesc with FP32 weights on GoogleNet, MobileNet-V1, MobileNet-V2, ResNet-101, ResNet-50, VGG16 and VGG19. We provide the instructions on enabling INT8 MKL-DNN quantization in Paddle Slim and show the results on above 7 models in accuracy.
+
+## 0. Install PaddlePaddle
+Follow PaddlePaddle [installation instruction](https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/image_classification/README_ngraph.md) to install PaddlePaddle. If you [build from source](https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/fluid/beginners_guide/install/compile/compile_Ubuntu_en.md), please use the following cmake arguments.
+
+```
+cmake .. -DCMAKE_BUILD_TYPE=Release -DWITH_GPU=OFF -DWITH_MKL=ON -DWITH_MKLDNN=ON  -DWITH_TESTING=ON  -WITH_FLUID_ONLY=ON  -DWITH_INFERENCE_API_TEST=ON -DON_INFER=ON -DWITH_SLIM_MKLDNN_FULL_TEST=ON
+```
+
+Note: MKL-DNN and MKL are required.
+
+## 1. Accuracy benchmark
+
+We provide the results of accuracy measured on Intel(R) Xeon(R) Gold 8280 on single core.
+
+   >**I. Top-1 Accuracy on Intel(R) Xeon(R) Gold 8280**
+
+| Model  | Dataset  | FP32 Accuracy  | INT8 Accuracy  | Accuracy Diff  |
+| :------------: | :------------: | :------------: | :------------: | :------------: |
+| GoogleNet    | Full ImageNet Val  |  70.50% |  70.20% |  0.30% |
+| MobileNet-V1 | Full ImageNet Val  |  70.78% |  70.36% |  0.42% |
+| MobileNet-V2 | Full ImageNet Val  |  71.90% |  71.57% |  0.33% |
+| ResNet-101   | Full ImageNet Val  |  77.50% |  77.53% |  -0.03%|
+| ResNet-50    | Full ImageNet Val  |  76.63% |  76.48% |  0.15% |
+| VGG16        | Full ImageNet Val  |  72.08% |  72.01% |  0.07% |
+| VGG19        | Full ImageNet Val  |  72.56% |  72.56% |  0.00% |
+
+Notes:
+* CPU turbo off.
+
+## 2. Commands to reproduce the above accuracy benchmark
+* #### Full dataset (Single core)
+   * ##### Download full ImageNet Validation Dataset
+```bash
+cd /PATH/TO/PADDLE/build
+python ../paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py
+```
+The converted data binary file is saved by default in ~/.cache/paddle/dataset/int8/download/int8_full_val.bin
+
+   * ##### GoogleNet Full dataset benchmark
+```bash
+cd /PATH/TO/PADDLE/build/python/paddle/fluid/contrib/slim/tests/
+python ./test_mkldnn_int8_quantization_strategy.py --infer_model /PATH/TO/PADDLE/build/third_party/inference_demo/int8v2/googlenet/model --infer_data /PATH/TO/CONVERTED/int8_full_val.bin --warmup_batch_size 100 --batch_size 1
```

Comment:
/PATH/TO/PADDLE is easy to understand, i.e. root path of Paddle git. However, it's difficult for user to know /PATH/TO/CONVERTED/. After searching in my .cache, I know it's ~/.cache/dataset/int8/download/.
This is not uesr friendly.